index.js

```diff
@@ -2,11 +2,6 @@
 
 var assign = require('object.assign');
 
-var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
-  strictMode: true, // add "use strict"
```

Comment:
I'm concerned about removing these strict mode settings; I'd prefer to be explicit, regardless of the defaults.

Reply:
I've used @loganfsmyth's suggestion [here](https://github.com/airbnb/babel-preset-airbnb/pull/37#discussion_r221808662), which seems to make the `transform-modules-commonjs` plugin unnecessary.

Should I add it back (with the defaults)?

Reply:
I'm fine with the changes in general, but I don't want to rely on defaults, and I'm very concerned with ensuring that `'use strict'` appears in any module code's transpiler output.

Reply:
That's be a massive breaking change on Babel's part. If this is truly a concern I'm not sure how you'd use any semver codebase.

Reply:
Hmm, I'd have to see an example. As far back as that option existed, it had to explicitly be set `false` to have any effect: https://github.com/babel/babel/commit/5802e2e334770d0c2e879b4e42acb63348573d92

In Babel 7, if something is treated as ESM, which all files are by default, they will get a `use strict`.